src/app/return-credentials/roc-return/roc-return.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="container container-bg-color">
+  <div class="card card-primary">
+    <div class="card-header card-header-style">ROC</div>
+    <div class="card-block">
+      <form [formGroup]="rocReturnForm" (ngSubmit)="addRocReturnInfo()">
```

Comment:
https://stackoverflow.com/questions/39831740/why-using-ngsubmit-instead-of-onsubmit 
check this one, I think we should use onSubmit() instead of ngSumit, thoughts ?

Reply:
same with other forms as well

Reply:
No need for now. Will explain on phone